src/SFA.DAS.LevyTransferMatching/Application/Queries/Pledges/GetApplications/GetApplicationsQueryResult.cs

```diff
@@ -88,13 +89,11 @@ public class Application
                         Id = o.Id,
                         PledgeLocationId = o.PledgeLocationId
                     }),
-                    IsJobRoleMatch = !pledgeResponse.JobRoles.Any() ||
-                                     roles.Any(r => r.Description == application.StandardRoute),
-                    IsLevelMatch = !pledgeResponse.Levels.Any() || pledgeResponse.Levels
-                        .Select(x => char.GetNumericValue(x.Last())).Contains(application.StandardLevel),
-                    IsLocationMatch = !pledgeResponse.Locations.Any() || application.Locations.Any(),
-                    IsSectorMatch = !pledgeResponse.Sectors.Any() ||
-                                    application.Sectors.Any(x => pledgeResponse.Sectors.Contains(x)),
+                    MatchJobRole = application.MatchJobRole,
```

Comment:
As this is the result for `GetApplications` (plural, rather than singular), you should not change it as part of your ticket - you should limit the scope to endpoints actually used by the page you are working on, unless you agree it with other devs beforehand. Other tickets are resulting in this class being refactored, and this will cause conflicts for you or someone else sooner or later.

Reply:
I have un-done any extra changes out of scope of this JIRA